packages/amplication-client/src/Application/git/GitOrganizations/GitOrganizations.tsx

```diff
@@ -0,0 +1,66 @@
+import { NetworkStatus } from "@apollo/client";
+import {
+    Button,
+  CircularProgress,
+  EnumButtonStyle,
+  Tooltip,
+} from "@amplication/design-system";
+import React, { useCallback } from "react";
+import useGetGitOrganizations from "../hooks/useGetGitOrganizations";
+
+const CLASS_NAME = "git-repos";
+
+type Props = {
+  workspaceId: string;
+  setSelectedGitOrganization:(gitOrganizationId:string)=>void
+};
+
+function GitOrganizations({ workspaceId ,setSelectedGitOrganization}: Props) {
+     const {
+    refetch,
+    gitOrganizations,
+    loading: loadingRepos,
+    networkStatus,
+  } = useGetGitOrganizations({
```

Comment:
I don't think it makes sense to create a hook for each server query. 
- There is no reuse of code here since it will probably not be used from anywhere else.
- The amount of code in this file is almost identical with or without the hook.. (instead of use ueGitOrganization, you just use useQuery) - the only code that was removed from this file is the actual gql...
- The name is not clear enough to allow anyone to ever re-use this hook
- putting gql in generic hooks breaks the benefit of GraphQL where each page can ask only for the data it needs. 
for example, a page need just the appId, and appName, and another page also need related entities with the app - so using a hook called "useApp" will eventually end up with a big query to cover all use cases where someone need an app with all fields and all relations.